gofakes3.go

```diff
@@ -779,6 +784,9 @@ func (g *GoFakeS3) copyObject(bucket, object string, meta map[string]string, w h
 	if srcObj.VersionID != "" {
 		w.Header().Set("x-amz-version-id", string(srcObj.VersionID))
 	}
+	if result.ETag != "" {
+		w.Header().Set("ETag", result.ETag)
```

Comment:
This is inconsistent with the other change. Here we don't surround the etag with `""` while we do above.